src/click/utils.py

```diff
@@ -57,6 +57,12 @@ def make_default_short_help(help, max_length=45):
     result = []
     done = False
 
+    if words[0] == "\b":
```

Comment:
The only valid use case for `\b` in Click is a single character on its own line, and for short help only the first line is relevant (subsequent paragraphs are already discarded). So this can be simplified to

```python
if words[0] == "\b":
    words = words[1:]
```

Reply:
I'll update it.